### PR TITLE
VEN-431 | Feature/deprecate button

### PR DESCRIPTION
**Pull request by Zeukkari**
- Deprecates our Button component in favor of HDS
- Deprecates our Icon component in favor of HDS
- Adds new components IconOutline and NavLink
- This PR could be cleaned up a bit. The SVG files could be removed and IconOutline component is defined a couple of times.

Screenshot from 2020-02-06 17-26-32

### CODE REVIEW

**Comment by Nizar-Rahme** [comment]
image
These new buttons don't look right!

File: src/common/internalLink/NavLink.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      <button
+        type="button"
+        onClick={onClick}
+        className={classNames(
+          styles.button,
+          styles[color],
+          styles.hasIcon,
+          className
+        )}
+      >
```
Comment (Nizar-Rahme) [inline src/common/internalLink/NavLink.tsx]: Why do we need a button here inside the link?

File: src/common/internalLink/internalLink.module.scss [hunk 2]
```diff
@@ -5,6 +5,13 @@
 
   text-decoration: none;
   color: var(--vene-internal-link-color);
+
+  display: flex;
+  align-items: center;
+
+  & > :first-child {
+    margin-right: 1rem;
```
Comment (Nizar-Rahme) [inline src/common/internalLink/internalLink.module.scss]: Can you use the spacing units we're using in other places to keep the interface consistent?

**Comment by Zeukkari** [comment]
> Is this PR for deprecating the Button **and** the Icon?

Yes. Please see VEN-450 for further information.

File: src/common/internalLink/NavLink.tsx (hunk 1 above)
Comment (Zeukkari) [reply src/common/internalLink/NavLink.tsx]: We don't, button has been removed.

File: src/common/internalLink/internalLink.module.scss (hunk 2 above)
Comment (Zeukkari) [reply src/common/internalLink/internalLink.module.scss]: using spacing units here

**Comment by Zeukkari** [comment]
Merging this now. I split leftover work into VEN-456 and VEN-457.

**Pull request merged**